prerender/cache/disk.py

```diff
@@ -26,3 +27,7 @@ def __init__(self) -> None:
     def set(self, key: str, payload: bytes, ttl: int = None, format: str = 'html') -> None:
         compressed = lzma.compress(payload)
         self._cache.set(key + format, compressed, expire=ttl)
+
+    async def modified_since(self, key: str, format: str = 'html') -> Optional[float]:
+        stats = await stat(key + format)
```

Comment:
It's not normal file here, the DiskCache class is actually a thin wrapper over [diskcache](https://pypi.python.org/pypi/diskcache).